### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by mingchuno**
Enjoy!

### CODE REVIEW

File: content/de/openletter/index.md [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
 templateKey: "open-letter-page"
 overrideSiteMeta:
-  title: An Open Letter by Hongkongers
-  description: "This is an urgent appeal from Hong Kong. The clock is ticking—we need your help."
+  title: Ein offener Brief von Hongkonger
+  description: "Dies ist ein dringender Aufruf aus Hongkong. Die Uhr tickt - wir brauchen Ihre Hilfe!"
   keywords:
     - ""
-title: "An Open Letter by Hongkongers"
+title: "Ein offener Brief von Hongkonger"
```
Comment (serious6) [inline content/de/openletter/index.md]: Suggestion: `Ein offener Brief der Einwohner Hongkongs`

File: content/de/openletter/index.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+Bitte fordern Sie Ihre Regierung nachdrücklich auf, uns Hongkongern bei der um sich greifenden Gewalt, der wir durch Hongkongs Regierung ausgesetzt sind, beizustehen. Obwohl wir um unsere Sicherheit fürchten, werden wir weiter für Demokratie und Freiheit kämpfen.
 
-Amid tear gas and rubber bullets, this once vibrant and safe metropolis is at a crossroads. Since the protests against the controversial extradition bill started in June, Hong Kong’s autonomy and freedom have been eroded beyond recognition. This is the ugly truth that the Hong Kong government does not want you to know: Hong Kong is becoming a police state.
+nmitten eines Hagels aus Tränengas und Geschossen ist nicht mehr viel von der einst pulsierenden und sicheren Metropole, die wir alle kannten, übrig geblieben. Seit die Proteste gegen das umstrittene Auslieferungsabkommen im Juni dieses Jahres begannen, wurde das hohe Maß an Autonomie und Freiheit bis zur Unkenntlichkeit zerfressen. Die bittere Wahrheit, die Hongkongs Regierung verschweigt, ist: Hongkong steht kurz davor, ein Polizeistaat zu werden.
```
Comment (serious6) [inline content/de/openletter/index.md]: `nmitten` -> **I**nmitten

File: content/de/openletter/index.md [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
+Anstatt die versprochenen politische Reformen umzusetzen, wurde die Regierung Hongkongs zu einem Unterdrückungsapparat. Polizeigewalt, sowohl von Hongkongs, als auch von Chinas Regierung befürwortet, sind leider Teil unseres täglichen Lebens geworden. Abgesehen von der Entmenschlichung der Protestierenden zu „Kakerlaken“ und den brutalen Angriffen gegen sie zum Zwecke der „Lenkung von Menschenmassen“, prügeln Polizeibeamte jetzt wahllos auf Fußgänger, Journalisten und medizinisches Personal ein. Polizeistationen werden immer dann geschlossen, wenn angeheuerte Schlägertrupps auf gewöhnliche Bürger losgehen, um abweichende Meinungen zu unterbinden. Verhaftungen und Verurteilungen werden immer willkürlicher. All dies sind Taktiken, die üblicherweise von einer nicht gewählten Regierung angewendet werden, um die Opposition zum Schweigen zu bringen. 
 
 
 
 * * *
 
-We are fighting for survival—and you can make a difference. Please:
+Die Bürger Hongkongs sind unter ihrer eigenen Regierung nicht mehr sicher. Uns läuft die Zeit davon - aber Sie können das ändern. Bitte stehen Sie uns bei:
```
Comment (serious6) [inline content/de/openletter/index.md]: The context of the sentence has been extended.

**Comment by mingchuno** [comment]
@serious6 we do have a platform for the translation and this PR is auto-generate. https://crwd.in/freedomhkg Would you mind sign up an account there and make the suggestion there?

**Pull request merged**